R/auth.R

```diff
@@ -16,8 +16,12 @@
 #' }
 #' @export
 discover <- function(auth_server) {
-  openid_config_url <- auth_server
-  urltools::path(openid_config_url) <- ".well-known/openid-configuration"
+  # ensure url ends with a single slash
+  auth_server_no_slash <- gsub("/$", "", auth_server)
+  auth_server <- paste0(auth_server_no_slash, "/")
+
+  # retrieve configuration
+  openid_config_url <- paste0(auth_server, ".well-known/openid-configuration")
```

Comment:
This is the endpoint right? https://auth1.molgenis.net/realms/Armadillo/.well-known/openid-configuration

Reply:
@christianhilbrands correct 👍